openshift/system/config/cache_store.yml

```diff
@@ -1,7 +1,3 @@
-base: &default
+production:
   - :mem_cache_store
-  - <%= ENV['MEMCACHE_SERVERS'].presence || 'system-memcache:11211' %>
-
-preview: *default
-
-production: *default
+  - <%= ENV.fetch('MEMCACHE_SERVERS', 'system-memcache:11211') %>
```

Comment:
Original version seems better as it will ignore empty `MEMCACHE_SERVERS` variable. Or is this intentional?

Reply:
Yes, good catch, you're right.